components/style/build.rs

```diff
@@ -62,3 +71,11 @@ fn main() {
         exit(1)
     }
 }
+
+fn main() {
+    println!("cargo:rerun-if-changed=build.rs");
+    generate_properties();
```

Comment:
We need to also be able to trigger a rebuild if the contents of the objdir change, which is a bit harder to do here.

Perhaps make the build script unconditional and have it detect this?